nansat/tests/test_nansat.py

```diff
@@ -748,22 +748,15 @@ def test_get_transect_data(self):
         self.assertEqual(type(t['lat']), np.ndarray)
         self.assertEqual(type(t['lon']), np.ndarray)
 
-    @unittest.skipUnless(MATPLOTLIB_IS_INSTALLED and 'DISPLAY' in os.environ, 'Matplotlib is required')
-    def test_digitize_points(self):
-        ''' shall return empty array in non interactive mode '''
-        for backend in matplotlib.rcsetup.interactive_bk:
-            # Find a supported interactive backend
-            try:
-                plt.switch_backend(backend)
-                break;
-            except:
-                pass
-        plt.ion()
+    @patch('nansat.nansat.PointBrowser')
+    def test_digitize_points(self, mock_PointBrowser):
+        ''' shall create PointBrowser and call PointBrowser.get_points() '''
+        value = 'ponits'
+        mock_PointBrowser().get_points.return_value = value
         n1 = Nansat(self.test_file_gcps, log_level=40)
```

Comment:
Why it's called n1? Could you name it n, or even more representative? n1 makes me think that this is not usual nansat object.
Why should we write a log of the tests? If that was used for a debugging than it shall be removed from production